### PR TITLE
removed TEST_USER buckets, added mocking to tests

### PR DESCRIPTION
**Pull request by amarjandu**
PR for removing TEST checkout buckets from cache_flow, added mocking for testing.

### CODE REVIEW

File: dss/storage/checkout/cache_flow.py [hunk 1]
```diff
@@ -19,8 +19,8 @@
 
 
 def is_dss_bucket(dst_bucket: str):
-    return dst_bucket in (os.environ['DSS_GS_CHECKOUT_BUCKET_TEST'], os.environ['DSS_S3_CHECKOUT_BUCKET_TEST'],
-                          os.environ['DSS_S3_CHECKOUT_BUCKET'], os.environ['DSS_GS_CHECKOUT_BUCKET'])
+    """Lambdas only have access to the standard checkout buckets, should not modify non-DSS buckets"""
```
Comment (DailyDreaming) [inline dss/storage/checkout/cache_flow.py]: This docstring seems out of context (what the function does and why), is this function necessarily directly associated with the lambdas?

File: tests/test_checkout_caching.py [hunk 2]
```diff
@@ -39,24 +40,32 @@ def tearDownClass(cls):
 
     def setUp(self):
         Config.set_config(BucketConfig.TEST)
+        self.aws_checkout_bucket = os.environ['DSS_S3_CHECKOUT_BUCKET']
```
Comment (DailyDreaming) [inline tests/test_checkout_caching.py]: Should these be test buckets?

File: tests/test_checkout_caching.py [hunk 3]
```diff
@@ -66,15 +75,15 @@ def test_aws_user_checkout_doesnt_create_tag(self):
         pass
```
Comment (DailyDreaming) [inline tests/test_checkout_caching.py]: `pass`?  Must have missed this before.  Probably should be taken out.

File: tests/test_checkout_caching.py (hunk 2 above)
Comment (amarjandu) [reply tests/test_checkout_caching.py]: removed, not needed

File: tests/test_checkout_caching.py (hunk 3 above)
Comment (amarjandu) [reply tests/test_checkout_caching.py]: removed all references in file

**Comment by DailyDreaming** [comment]
LGTM.

**Pull request merged**